contracts/cw3-fixed-multisig/src/state.rs

```diff
@@ -32,12 +32,15 @@ pub struct Proposal {
 
 impl Proposal {
     /// TODO: we should get the current BlockInfo and then we can determine this a bit better
-    pub fn current_status(&self) -> Status {
+    pub fn current_status(&self, block: &BlockInfo) -> Status {
         let mut status = self.status;
 
-        // if open, check if voting is passed on timed out
+        // if open, check if voting is passed or timed out
         if status == Status::Open && self.yes_weight >= self.required_weight {
-            status = Status::Passed
+            status = Status::Passed;
+        }
```

Comment:
Nice. No need to use `else if` here. Let's just not change the order of these checks.